opencensus.go

```diff
@@ -186,9 +186,10 @@ type ZipkinConfig struct {
 }
 
 type JaegerConfig struct {
-	Endpoint       string `json:"endpoint"`
-	ServiceName    string `json:"service_name"`
-	BufferMaxCount int    `json:"buffer_max_count"`
+	AgentEndpoint     string `json:"agent_endpoint"`
+	CollectorEndpoint string `json:"collector_endpoint"`
```

Comment:
To not break older configurations, we should maintain the `endpoint` parameter as is.